modules/introduction/pages/whats-new.adoc

```diff
@@ -1,193 +1,108 @@
 
-For more information, see xref:n1ql:n1ql-language-reference/userfun.adoc[User-Defined Functions].
+To learn more, see xref:install:non-root.adoc[Non-Root Install and Upgrade].
 
-==== Analytics Enhancements
+==== Data Import via the UI
 
-===== Window Functions
+The Couchbase Web Console provides a graphical interface to import data in JSON, CSV, or TSV formats. While the command-line utilities are recommended for high-performance data import and for importing large datasets, the graphical interface is enables quick data ingestion for development or testing.
 
-Window functions are also available in the Analytics service. While the syntax and semantics are portable between query and analytics services, the Query service runs in SMP mode while the Analytics service runs in an MPP architecture.
+To learn more about importing data from the UI, see xref:manage:import-documents/import-documents.adoc[Import Documents].
 
-For more information, see xref:analytics:8_builtin.adoc#WindowFunctions[Window Functions].
+==== Backup to AWS S3 or any S3 Compatible Object Store
 
-===== Scan Consistency in Analytics
+The `cbbackupmgr` utility adds support for object store and enables you to directly back up to and restore from AWS S3 or any other S3 compatible object store. 
 
-With scan consistency in Analytics, cluster operations like rebalance and failover will not affect the business-critical reports and dashboards that are being used to glean insights and drive business decisions.
+With the ability to backup large datasets directly to the cloud, you can leverage cloud-based object stores for your disaster recovery or archival use cases, or build containerized applications.  
 
-For more informaiton, see xref:analytics:query-params.adoc[Query Parameters].
+To learn more about this feature, see xref:backup-restore:cbbackupmgr-backup.adoc#cloud-integration[cbbackupmgr backup] and xref:backup-restore:cbbackupmgr-restore.adoc#cloud-integration[cbbackupmgr restore].
 
-===== Storage compression
+==== Backup Info Command
 
-Storage compression using https://en.wikipedia.org/wiki/Snappy_(compression)[snappy] enables a reduction of the disk footprint of the Analytics Service resulting in reduced storage costs. Compression can be enabled per dataset or globally for all newly created datasets.
+The `cbbackupmgr info` command provides additional information such as type of backup, start and end dates for the merged backups, and number of views, global secondary indexes (GSI), or FTS indexes. You can obtain the output in JSON format for automation purposes. 
 
-For more information, see xref:analytics:5_ddl.adoc[Data Definition Language (DDL)].
+To learn more about the command, see xref:backup-restore:cbbackupmgr-info.adoc[cbbackupmgr info].
```

Comment:
```suggestion
To learn more about the enhancements to this command, see xref:backup-restore:cbbackupmgr-info.adoc[cbbackupmgr info].
```

Reply:
This was added in 6.5.1, but is being highlighted now. So would prefer to leave the desc as is.